Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:carbon
+FROM node:carbon-alpine
```

Comment:
I don't know if this will implicate somehow on how app runs on production env 🤔

Reply:
Many packages need gcc, g++ and other things for compile the binary code. This will not work fine with carbon-alpine.

Reply:
@PauloPaquielli remove those changes about docker image... stick to the changes related with issue #23

Reply:
Done @rafaelverger @RodolfoSilva. Sorry for the delayed response :)